library/alloc/src/sync.rs

```diff
@@ -84,9 +84,26 @@ macro_rules! acquire {
 ///
 /// Shared references in Rust disallow mutation by default, and `Arc` is no
 /// exception: you cannot generally obtain a mutable reference to something
-/// inside an `Arc`. If you need to mutate through an `Arc`, use
-/// [`Mutex`][mutex], [`RwLock`][rwlock], or one of the [`Atomic`][atomic]
-/// types.
+/// inside an `Arc`. If you need to mutate through an `Arc`, you have several options:
```

Comment:
I think this could connect better with the previous sentence if this is phrased more as an exception to the previous statement:
```suggestion
/// inside an `Arc`. If you do need to mutate through an `Arc`, you have several options:
```